From ab522db34086146535731b9c485be45edd5f4b2a Mon Sep 17 00:00:00 2001
From: William <william@undefined.re>
Date: Thu, 25 Feb 2021 13:04:53 +0100
Subject: [PATCH] tuareg: disable merlin-company-backend

---
 modules/lang/ocaml/config.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/lang/ocaml/config.el b/modules/lang/ocaml/config.el
index 60c19ade3..9fcb17b54 100644
--- a/modules/lang/ocaml/config.el
+++ b/modules/lang/ocaml/config.el
@@ -50,7 +50,7 @@
       (merlin-mode)))
 
   (after! tuareg
-    (set-company-backend! 'tuareg-mode 'merlin-company-backend)
+    ;(set-company-backend! 'tuareg-mode 'merlin-company-backend)
     (set-lookup-handlers! 'tuareg-mode :async t
       :definition #'merlin-locate
       :references #'merlin-occurrences
-- 
2.30.1

