From f4c7c6888b558f4e433af6a9accad21dd5381cb6 Mon Sep 17 00:00:00 2001
From: William <william@undefined.re>
Date: Tue, 4 May 2021 19:27:38 +0200
Subject: [PATCH] leave-ret-alone2

---
 modules/lang/org/config.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/lang/org/config.el b/modules/lang/org/config.el
index 79afb071c..54d7bade6 100644
--- a/modules/lang/org/config.el
+++ b/modules/lang/org/config.el
@@ -1046,8 +1046,8 @@ compelling reason, so..."
             :ni CSup      #'org-shiftup
             :ni CSdown    #'org-shiftdown
             ;; more intuitive RET keybinds
-            :n [return]   #'+org/dwim-at-point
-            :n "RET"      #'+org/dwim-at-point
+            ;;:n [return]   #'+org/dwim-at-point
+            ;;:n "RET"      #'+org/dwim-at-point
             :i [return]   (cmd! (org-return electric-indent-mode))
             :i "RET"      (cmd! (org-return electric-indent-mode))
             :i [S-return] #'+org/shift-return
-- 
2.31.1

